doc/man1/s_server.pod

```diff
@@ -125,14 +130,34 @@ manual page.
 
 Print out a usage message.
 
-=item B<-accept port>
+=item B<-port port>
```

Comment:
s_server's -port argument was present but undocumented (even in usage()) in SSLeay 0.8.1b, the earliest point in this git repo.  One might wonder whether it was intentionally undocumented.